ReactWindows/ReactNative/Bridge/JavaScriptBundleLoader.cs

```diff
@@ -86,9 +86,9 @@ public override string SourceUrl
             {
                 try
                 {
-                    var storageFile = SourceUrl.StartsWith("ms-appx:///")
-                        ? await StorageFile.GetFileFromApplicationUriAsync(new Uri(SourceUrl))
-                        : await StorageFile.GetFileFromPathAsync(SourceUrl);
+                    var storageFile = SourceUrl.StartsWith("file:///")
```

Comment:
Note that this breaks when running on PC, where file URLs don't have the "file:///" prefix.